submissions/modupeAdeOnojobi/dom/contact.js

```diff
@@ -0,0 +1,56 @@
+let contactForm = document.getElementById('contactForm');
```

Comment:
How come you are doing validation for contact form and your task says newsletter form validation

Reply:
Kindly do one task at a time. Don't do two tasks at the same time hence you won't meet the success criteria for your PR to be merged

Reply:
You have to remove this file